### ISSUE TITLE
force:apex:test:run --verbose no longer emitting output to stdout

### ISSUE THREAD

**Issue by tfuda** [forcedotcom/salesforcedx-vscode#3506]
Since upgrading to sfdx-cli version 7.112.0, I am noticing that the following command is no longer outputting command output to stdout:

```sfdx force:apex:test:run -w 60 --verbose```

Prior to this installing this upgrade, I would see periodic output to stdout indicating the cli was polling for updates, and then at the end of the test run, it would spit out a summary of all of the tests that were run, along with a summary of how many tests passed or failed. The lack of output is particularly problematic in our CI environment, which runs against Scratch Orgs, and then deletes the Scratch Orgs upon completion, so there's no record of the tests that failed.

### Steps To Reproduce:

1. Execute the command against your Apex code base with unit test classes: 

```sfdx force:apex:test:run -w 60 --verbose```

### Expected result

I expect to see periodic progress updates while the tests are running, and a test summary when the test run completes.

### Actual result

No output is emitted to stdout either while the tests are running, or once the tests have completed.

### System Information

```
D:\Development\sfptdev.tom>sfdx version --verbose --json
{
        "cliVersion": "sfdx-cli/7.113.0",
        "architecture": "win32-x64",
        "nodeVersion": "node-v14.17.4",
        "pluginVersions": [
                "@oclif/plugin-autocomplete 0.3.0 (core)",
                "@oclif/plugin-commands 1.3.0 (core)",
                "@oclif/plugin-help 3.2.2 (core)",
                "@oclif/plugin-not-found 1.2.4 (core)",
                "@oclif/plugin-plugins 1.10.1 (core)",
                "@oclif/plugin-update 1.4.0-3 (core)",
                "@oclif/plugin-warn-if-update-available 1.7.0 (core)",
                "@oclif/plugin-which 1.0.3 (core)",
                "@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)",
                "@salesforce/sfdx-trust 3.6.0 (core)",
                "alias 1.1.10 (core)",
                "apex 0.2.3 (core)",
                "auth 1.7.1 (core)",
                "config 1.2.23 (core)",
                "custom-metadata 1.0.12 (core)",
                "data 0.6.0 (core)",
                "etcopydata 0.6.4-Beta (beta)",
                "generator 1.1.7 (core)",
                "limits 1.2.1 (core)",
                "org 1.6.9 (core)",
                "salesforce-alm 52.2.4 (core)",
                "schema 1.0.8 (core)",
                "sfdx-cli 7.113.0 (core)",
                "source 1.0.7 (core)",
                "telemetry 1.2.3 (core)",
                "templates 52.1.0 (core)",
                "user 1.4.0 (core)"
        ],
        "osVersion": "Windows_NT 10.0.19042"
}
```

This also happens in our CI/CD environment, which is based on GitHub Actions running on Ubuntu VMs.

**Comment by tfuda** [comment]
Sorry @WillieRuemmele, I don't know how the hell I ended up filing this in the salesforcedx-vscode issue tracker. That was not my intention. Thank you for transferring it, but I don't it in the forcedotcom/cli issue tracker. Should I just file a new issue in forcedotcom/cli?

**Comment by tfuda** [comment]
Wait... I misread your comment. Why did you transfer this issue? It has nothing to do with vscode. This is a sfdx-cli issue. Please transfer this back to forcedotcom/cli

**Comment by WillieRuemmele** [comment]
yes. I understand the confusion. The team that owns the `apex:*` command, also owns the VSCode extensions and prefers to track issues here together

**Comment by tfuda** [comment]
Understood

**Comment by smaddox-sf** [comment]
internal note: might be related to #3172

**Comment by AnanyaJha** [comment]
Hi @tfuda thanks for reporting this! Were you seeing this work correctly on a previous version of the CLI? If so, what version & could you share the output of `sfdx plugins --core`?

**Comment by tfuda** [comment]
Up until some time last week, our CI platform was using an old version of sfdx-cli (7.82.0) and the test output was being written to the console. But then we modified our GitHub workflow to use the stable Linux distrubtion from https://developer.salesforce.com/media/salesforce-cli/sfdx/channels/stable/sfdx-linux-x64.tar.xz and that's when we observed that nothing is being written to the console any more. I used the npm distribution to install 7.82.0 locally on my Windows machine and I observe the same thing; the test progress and summary are written to the Windows command window. So, basically, this is the output of "sfdx plugins --core" for the last known version that behaves as expected:

**Comment by daveespo** [comment]
This sounds like a duplicate of https://github.com/forcedotcom/salesforcedx-apex/issues/211 which goes back to mid-June (sfdx-cli/7.107.0)

**Comment by tfuda** [comment]
Based on information coming from the issue that @daveespo mentions above, we tried add the "-r human" option. This had the desired effect of restoring the test summary information to stdout, but not the periodic messages that are output while the tests are running. IMO, these aren't all that informative anyway, other than providing an indication that the tests are executing in the background.

**Comment by jeffb-sfdc** [comment]
I'll take this on.
